ReactiveCocoa/Objective-C/RACCommand.h

```diff
@@ -24,7 +24,7 @@ extern NSString * const RACUnderlyingCommandErrorKey;
 
 /// A command is a signal triggered in response to some action, typically
 /// UI-related.
-@interface RACCommand : NSObject
+@interface RACCommand<__covariant InputType> : NSObject
```

Comment:
@mdiep Great point! I was unaware that multiple variance keywords were even available. After reading up on this the suggested change definitely makes sense. Strange that this is available in Obj-C but not in Swift. Thanks!